.travis.yml

```diff
@@ -17,15 +16,6 @@ env:
   - DJANGO=2.1
   - DJANGO=2.2
```

Comment:
Would be nice to try newer versions of django and see whether or not we're compatible, but I guess that probably shouldn't be done right now.